### PR TITLE
Portfolio: Desktop version PR

### PR DESCRIPTION
**Pull request by Sonikak004**
### In this Branch I have : 💯 

- Used **_HTML_** to create Desktop version 👍 
- Used **_CSS_** to style the Desktop Version ✅   
- Used _Media Queries_ 😄 
- Linters Checked ✔️ 

`Tried to Replicate Figma Template 😜 `

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -892,3 +909,601 @@ input {
...
+
+  header {
+    background: #3c3a39;
+  }
+
+  .items {
+    background-color: #3c3a39;
+  }
+
+  .nav-icon {
```
Comment (Kossi-stack) [inline style.css:942]: - [ ] Kindly make the background image of the header width 100%
- [ ] Your Design
Screenshot 2023-01-21 at 12 28 04



- [ ] Figma
Screenshot 2023-01-21 at 12 33 31

File: style.css [hunk 2]
```diff
@@ -892,3 +909,601 @@ input {
...
+    top: -2314px;
+    left: 630px;
+    height: 345px;
+    width: 357px;
+    background-size: 68%;
+    background-image: url(./images/card3.png);
+  }
+
+  .about-me {
+    width: 100%;
```
Comment (Kossi-stack) [inline style.css:1260]: - [ ] Kindly fix the background image 
- [ ] Your design
Screenshot 2023-01-21 at 12 38 25
- [ ] Figma
Screenshot 2023-01-21 at 12 45 16

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Sonikak004,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

- [ ] Kindly fix your twitter social handle.

 _Check the comments under the review for more changes_.

## Optional suggestions

 - [OPTIONAL] Kindly edit the PR's title to something more descriptive. it can be " Portfolio: Desktop version  PR".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (joskalenda) [inline style.css:942]: As requested by the previous reviewer, please make the necessary changes to match the original design, see this record below how your page behaves on both mobile and desktop version

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Sonikak004 , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Good readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Sonikak004 

While you have done a good job implementing the project requirements. But there are few things that you still need to work on to complete this milestone. But I know you can easily conquer this one. Right ? 🎯😄
**_You are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Descriptive PR
- ✅ Good README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

From [your latest commit](https://github.com/Sonikak004/my-portfolio/pull/6/commits/6026b9a1de2e8a5afc63f5911595feb6bb4efd7e) I can see that changes requested by previous reviewer are not implemented. No worries, let's implement them now 💪 You got this. Let's fix them so that we can move to the next milestone of the project. What do you say ? 🎯
_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @Sonikak004 🙋‍♂️

Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on the design practice.

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
- [ ] [OPTIONAL] I have also noticed that you are using much of the `margins` in your portfolio. I applaud 👏 👏  you for the great effort to make the design look incredible. However, if not properly used it can be [harmful](https://news.ycombinator.com/item?id=22676442) and cause horizontal scrolls. Kindly verify the margins you used to ensure that they don't cause any harm to your portfolio.
- [ ] [OPTIONAL] The sections always adapt to 100$ width of the html, therefore unless the width of the html has been changed you may consider using width: 100% on sections. As you are fixing the horizontal scroll effects, Kindly consider verifying the widths and margins. Read more [here](https://polypane.app/blog/strategies-for-dealing-with-horizontal-overflows/) and [here](https://www.quora.com/Is-setting-negative-margins-in-CSS-to-be-considered-bad-practice)

Your portifolio demo

https://user-images.githubusercontent.com/99394374/215254604-f9acfe01-d464-4092-aa62-d3c300b59256.mp4


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: style.css [hunk 3]
```diff
@@ -312,6 +321,8 @@ button {
 /* professional art cards */
 
 .professional-art {
+  left: -21px;
+  position: relative;
   display: grid;
```
[the remark this answers is not in the document]
Comment (zieeco) [reply style.css:330]: - Hey, you've really done a great job, however as pointed out by the previous reviewer, your mobile design is still having the horizontal scroll effect and it's so obvious that I can not overlook it to approve your project. Please, kindly see the image below to fix this issue. I tried using a screencast, but it misbehaving that's why i did the screenshot. :pray: 



Screenshot from 2023-01-28 12-14-21
Screenshot from 2023-01-28 12-14-16
Screenshot from 2023-01-28 12-14-10
Screenshot from 2023-01-28 12-14-05
Screenshot from 2023-01-28 12-14-00
Screenshot from 2023-01-28 12-13-54
Screenshot from 2023-01-28 12-13-47
Screenshot from 2023-01-28 12-13-38
Screenshot from 2023-01-28 12-13-00

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Sonikak004 ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Sonikak004,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝


 ## Required Changes ♻️
- Please you fixed the horizontal overflowing issue as the previous reviewer requested. Kindly check the GIF below.
Screen Recording 2023-01-28 at 3 07 00 PM


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @Sonikak004👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well-written PR description ✔️ 
* All tests passed ✔️ 

## Optional suggestions:

- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" or "linters are added" ... etc 👍 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**